apps/api/src/app/controllers/__tests__/setupIntegrationTest.ts

```diff
@@ -0,0 +1,40 @@
+import { INestApplication } from '@nestjs/common';
```

Comment:
This should be called `setUpIntegrationTest`

Reply:
Good catch on the filename. I first thought you meant 'import { INestApplication as setUpIntegrationTest } from '@nestjs/common'.

Reply:
Renamed.